### ISSUE TITLE
resource/aws_fsx_windows_file_system: Increase default creation timeout

### ISSUE THREAD

**Issue by bflad** [hashicorp/terraform-provider-aws#16315]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

```
Terraform CLI v0.12.29

Terraform AWS Provider main branch
```

### Affected Resource(s)

* aws_fsx_windows_file_system

### Terraform Configuration Files

```hcl
resource "aws_fsx_windows_file_system" "test" {
  active_directory_id = aws_directory_service_directory.test.id
  skip_final_backup   = true
  storage_capacity    = 32
  subnet_ids          = [aws_subnet.test1.id]
  throughput_capacity = 8
}
```

### Debug Output

Can provide if needed.

### Expected Behavior

Resource creation happens without timeout error for majority of cases.

### Actual Behavior

Very frequently hitting the default creation timeout of 30 minutes:

```
=== CONT  TestAccAWSDataSyncLocationFsxWindows_basic
TestAccAWSDataSyncLocationFsxWindows_basic: resource_aws_datasync_location_fsx_windows_file_system_test.go:86: Step 1/2 error: Error running apply: 2020/11/19 09:57:15 [DEBUG] Using modified User-Agent: Terraform/0.12.29 HashiCorp-terraform-exec/0.10.0
Error: Error waiting for filesystem (fs-00b5e00e73764e811) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSDataSyncLocationFsxWindows_basic (4748.98s)

=== CONT  TestAccAWSDataSyncLocationFsxWindows_subdirectory
TestAccAWSDataSyncLocationFsxWindows_subdirectory: resource_aws_datasync_location_fsx_windows_file_system_test.go:147: Step 1/2 error: Error running apply:
Error: Error waiting for filesystem (fs-090dcf07d70912a72) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSDataSyncLocationFsxWindows_subdirectory (4186.00s)

=== CONT  TestAccAWSFsxWindowsFileSystem_StorageCapacity
TestAccAWSFsxWindowsFileSystem_StorageCapacity: resource_aws_fsx_windows_file_system_test.go:535: Step 1/3 error: Error running apply:
Error: Error waiting for filesystem (fs-0afc18f052141e201) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSFsxWindowsFileSystem_StorageCapacity (4006.30s)
```

Recommend bumping it to 45 or 60 minutes in the resource code and updating the documentation.

### Steps to Reproduce

1. `TF_ACC=1 go test -run='(TestAccAWSFsxWindowsFileSystem_|TestAccAWSDataSyncLocationFsxWindows_)' -parallel=20 -timeout=4h -v`

### Important Factoids

Could only be an issue when there are lots of parallel creations.

### References

* [Contributing Guide on Running/Writing Acceptance Testing](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md)

**Comment by denissimonovski** [comment]
@bflad I'm interested in working on this issue. I tried pushing the code to the repo but it gave me a "permission denied". This is my first ever PR for an open source project like this. What do I need to do to get the permissions?

**Comment by dthvt** [comment]
@denissimonovski you should fork this repo on github to your personal account.  Then clone your forked repo to your local machine and perform the work necessary on a new branch.  Push that branch to github, and then you can open a pull request on the github website which will notify the terraform developers that you would like to contribute code.  It will also kick off the automated CI pipeline which will run automated checks on the proposed code.

You may find this guide helpful if you aren't familiar with Github based development. https://guides.github.com/introduction/flow/

**Comment by denissimonovski** [comment]
I've submitted the PR (#16363) and thanks @dthvt for the pointers. :)

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.17.0 of the Terraform AWS Provider, later this week. Thank you @denissimonovski and @dthvt 👍
